app/notifications/send_notifications.py

```diff
@@ -0,0 +1,94 @@
+from flask import current_app
+from app.config import QueueNames
+from app.models import EMAIL_TYPE, KEY_TYPE_NORMAL, SMS_TYPE, Service, Template
+from app.notifications.process_notifications import (
+    persist_notification,
+    send_notification_to_queue,
+    send_to_queue_for_recipient_info_based_on_recipient_identifier
+)
+
+
+def send_notification_bypass_route(
+        service: Service,
+        template: Template,
+        notification_type: str,
+        recipient: str = None,
+        personalisation: dict = None,
+        sms_sender_id: str = None,
+        recipient_item: dict = None,
+        api_key_type: str = KEY_TYPE_NORMAL
+):
+    """
+    This will create a notification and add it to the proper celery queue using the given parameters
+
+    :param service: the service sending the notification
+    :param template: the template to use to send the notification
+    :param notification_type: the type of notification to send (sms or email)
+    :param recipient: the sms number or email address to send the notification to
+    :param personalisation: a dictionary of personalisation fields to include in the notification
+    :param sms_sender_id: the sms sender to use when sending an sms notification,
+        Note: uses service default for sms notifications if not passed in
+    :param recipient_item: a dictionary specifying 'id_type' and 'id_value'
+    :param api_key_type: the api key type to use, default: 'normal'
+    """
+
+    if recipient is None and recipient_item is None:
+        current_app.logger.critical(
+            'Programming error attempting to use send_notification_bypass_route, both recipient and recipient_item are '
+            'None. Please check the code calling this function to ensure one of these fields is populated properly.'
+        )
+
+        return
```

Comment:
This should raise a technical exception or something. We want to capture this as a critical failure because nothing should make it this far without a va profile id.

Reply:
now it raise NotificationTechnicalFailureException